### PR TITLE
pod: update a few documents for perlgov changes

### PR DESCRIPTION
**Pull request by rjbs**
Nothing groundbreaking here, just trying to nudge towards coherency.

### CODE REVIEW

File: pod/perlpolicy.pod [hunk 1]
```diff
@@ -24,22 +24,26 @@ some are actively patching their pet area (threads, Win32, the regexp
...
 
+Among these people are the core Perl team.  These are trusted volunteers
+involved in the ongoing development of the Perl language and interpreter.
+They are not required to be language developers or committers.
+
 Over this group of porters presides Larry Wall.  He has the final word
 in what does and does not change in any of the Perl programming languages.
 These days, Larry spends most of his time on Raku, while Perl 5 is
-shepherded by a "pumpking", a porter responsible for deciding what
+shepherded by a steering council of porter responsible for deciding what
```
Comment (Grinnz) [inline pod/perlpolicy.pod]: "of porters"

**Review by jkeenan — APPROVED** [review]
Once you've applied @Grinnz's correction, this LGTM

**Comment by Leont** [comment]
Agreed with the above

File: pod/perlpolicy.pod (hunk 1 above)
Comment (rjbs) [reply pod/perlpolicy.pod]: Thanks.

**Comment by rjbs** [comment]
(The email address mentioned in this commit doesn't exist yet, so at the very least I will be waiting for it to exist or get changed.  Also, wouldn't mind @neilb and @xsawyerx eyes on this!)

**Review by xsawyerx — APPROVED** [review]

**Pull request merged**